about.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone</title>
+    <link rel="stylesheet" href="about.css">
+    <script
+      src="https://kit.fontawesome.com/5d13e94c8b.js"
+      crossorigin="anonymous"
+    ></script>
+</head>
+<body>
+    <header class="header">
+        <div class="header-div">
+            <div>
+            <ul class="icons-ul">                
+                <li><img class="head-icon" src="./images/faceb.png" alt="facebookicon"></li>
+                <li><img class="head-icon" src="./images/twtr.png" alt="twitter-icon"></li>
```

Comment:
- [ ] Great work on adding "alt" text to these alt values.  However, he also requested you add "alt" text to all the `alt` properties on your project:

![image](https://user-images.githubusercontent.com/57421823/164949514-5bc1ad18-14ec-4573-b962-e69dd06e110d.png)

Please, write alt text for all your `img` tags.

Reply:
done

Reply:
done